projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -932,10 +932,11 @@ private function get_plugin_status( $plugin_slug ) {
 				break;
 			case 'mailpoet':
 				$response['needsConnection'] = true;
-				if ( class_exists( '\MailPoet\Config\ServicesChecker' ) ) {
```

Comment:
Should we always return ```$response['isConnected']```?  this would make it easier to deal with on the client side as a API consumer since you don't have to check if the value exists.